auth-web/src/stores/business.ts

```diff
@@ -268,6 +269,19 @@ export const useBusinessStore = defineStore('business', () => {
     }
   }
 
+  async function loadFiling () {
+    const filingID = ConfigHelper.getFromSession(SessionStorageKeys.FilingIdentifierKey)
+    state.filingID = filingID
+    const response = await BusinessService.searchFiling(filingID).catch(() => null)
+    if (response?.status === 200) {
+      ConfigHelper.addToSession(SessionStorageKeys.BusinessIdentifierKey, response?.data.filing.business.identifier)
+    } else if (response?.status === 404) {
+      throw Error('No match found for Filing Number')
```

Comment:
I'm not seeing this error if I search for a non-existent filing id.

Reply:
Looks good now. Thanks!

![image](https://github.com/user-attachments/assets/cbd81173-2b9a-425d-8cde-6b5b049dd4f5)